h2o-core/src/main/java/water/api/HeartbeatHandler.java

```diff
@@ -0,0 +1,27 @@
+package water.api;
+
+import water.H2O;
+import water.H2ONode;
+import water.api.schemas3.HeartbeatV3;
+
+class HeartbeatHandler extends Handler {
```

Comment:
is this all output info needed?

maybe we could simplify to as simple PingHandler :)

also, it would be great if this was actually implemented on SW side as an extension and the SW-specific jars were added to the runtime dynamically - IMHO that would help the codebase not just in this case but for other SW-specific extensions

Reply:
Yup, for sparkling water ping, we just need this info. We access also other endpoints, for for the ping this is sufficient. 

I can rename it :)

Regarding the extension -> I really like to idea! What about finalizing the major release with the current approach and then move SW specific endpoints toward sparkling water code-base with the run-time registration? We can create just new sub-module in SW where we define the extension points